### DISCUSSION TITLE
"Bug Basher" Skill Badge

### DISCUSSION THREAD

**Discussion by c-py** [sunflower-land/sunflower-land#1113] [Ideas]
### Details

Plenty of community members dive deep to find bugs in code and art but don't necessarily develop code or create art.

This proposal is for a "Bug Basher" badge, designed to award those who make a significant contribution at finding and resolving a bugs.

### Badge Bonus

I'm not sure what bonus would be applicable for a bug basher. Perhaps it could be something related to chickens?

---

I'd love to hear thoughts from everybody 🐜.

**Comment by JustJayHSP** [comment]
For the role of Bug Basher skill. 

If we're gonna use it on Chickens: 
a. I suggest having a chance to get twice the eggs like the Golem so its not the same as Mutant Chicken Buffs.

b. When harvesting an egg from a chicken there's a chance that the chicken's automatically fed without the consumption of Wheat.

c. Next would be a 10-15% more yield on Wheat for easier feeds

d. Other than that 5-10% yields more or faster cooldown

e. If we're gonna keep Cakes in the future they could have discount on their eggs when crafting

**Comment by JustJayHSP** [comment]
If we're not going for the Chicken Route Buff for Bug Basher Skill:

We could also do
a. More Seed Stocks increase it by 30% or 50% so they'd sync less

b. Increase chances on "Bonus Seeds" upon Harvest or if not increased chances why not give them an extra seed upon getting a Bonus Seed prompt

c. Gold Ore Deposits recover faster

I'll add more if I find something interesting

**Comment by c-py** [comment]
Thanks @JustJayHSP 🎉

**Comment by NotKeen** [comment]
Zmash the beetles! I love bug hunting. Maybe I'll earn this badge someday.

Reward: All boosts are multiplicative instead of additive 😅 (just kidding)

Maybe +5 % on all crop yield or something like that? Because no more bugs eating the crops, you know? 😉

Edit: Thought of another cool one, how about a tax reduction in the upcoming trading features?

**Reply by sir-canada** [reply]
> Edit: Thought of another cool one, how about a tax reduction in the upcoming trading features?

I like the last one :+1:
